acceptance/tests/language/binary_data_type.rb

```diff
@@ -0,0 +1,76 @@
+test_name 'C98346: Binary data type' do
+  require 'puppet/acceptance/puppet_type_test_tools.rb'
+  extend Puppet::Acceptance::PuppetTypeTestTools
+
+  app_type        = File.basename(__FILE__, '.*')
+  tmp_environment = mk_tmp_environment_with_teardown(master, app_type)
+  fq_tmp_environmentpath  = "#{environmentpath}/#{tmp_environment}"
```

Comment:
It's a nit.  Think I'd want to see it used consistently or not at all. This is a mix. And totally not the hill to die on.